src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin star($size) {
+  width: $size;
+  height: $size;
+}
+
+@mixin zero($size) {
```

Comment:
1. Not definitive names for mixins.
2. Don't set margins and paddings in mixins, it can break BEM rules and be missed.